dbservice/postgresdb/connect.go

```diff
@@ -2,16 +2,39 @@ package postgresdb
 
 import (
 	"errors"
+	"time"
 
+	"github.com/aquasecurity/postee/log"
 	"github.com/jmoiron/sqlx"
 )
 
+const (
+	CONN_RETRIES = 10
+)
+
+var dbConn *sqlx.DB
+
 var psqlConnect = func(connectUrl string) (*sqlx.DB, error) {
-	db, err := sqlx.Connect("postgres", connectUrl)
-	if err != nil {
-		return nil, err
+	if dbConn == nil {
```

Comment:
I think we could still have a race condition here. Maybe use a sync.Once object?

Reply:
Actually, we don't need here a singleton instance, it will happen only on the library initialization.